tests/test_amclient.py

```diff
@@ -502,6 +502,35 @@ def test_get_transfer_status_invalid_uuid(self):
                            "baad-f00d-8120-b1c51c2ab5db")
         assert message_type == 'transfer'
 
+    @vcr.use_cassette('fixtures/vcr_cassettes/ingest_status.yaml')
+    def test_get_ingest_status(self):
+        """Test the successful return of the status of an ingest for a
+        valid SIP UUID.
+        """
+        response = amclient.AMClient(
+            am_api_key=AM_API_KEY,
+            am_user_name=AM_USER_NAME,
+            am_url=AM_URL).get_ingest_status("23129471-09e3-467e-"
+                                             "88b6-eb4714afb5ac")
+        message = response['message']
+        message_type = response["type"]
+        assert message == ("Fetched status for 23129471-09e3-467e-"
+                           "88b6-eb4714afb5ac successfully.")
+        assert message_type == "SIP"
+
+    @vcr.use_cassette('fixtures/vcr_cassettes/ingest_status_invalid_uuid.yaml')
+    def test_get_ingest_status_invalid_uuid(self):
+        """Test the successful return of the status of an ingest for a
+        valid SIP UUID.
```

Comment:
I think this docstring is inaccurate. Aren't you testing for an invalid SIP UUID here?